src/shared/variable.shared.ts

```diff
@@ -0,0 +1,8 @@
+
+const url = process.env.BACKEND_URL;
+
+const headers = {
+  'Content-Type': 'application/json',
```

Comment:
Możesz to od razu do defaultów w axiosie przypisać

Reply:
dlaczego to zamknąłeś bez żadnej odpowiedzi ani poprawki?

Reply:
Poprawki są w pliku /boot/axios.ts. Zrobiłem defaultowy axios z url i headers.